torch/_inductor/fx_utils.py

```diff
@@ -87,7 +87,8 @@ def is_fake_tensor_same(new, old):
                 return all(
                     is_fake_tensor_same(new_i, old_i) for new_i, old_i in zip(new, old)
                 )
-            assert isinstance(new, torch.Tensor)
+            if not isinstance(new, torch.Tensor):
+                return new == old
```

Comment:
What types get to this line?

Reply:
I saw `SymInt`s get to this line.

Reply:
can you put an `assert isinstance(new, (torch.SymInt, torch.SymBool, torch.SymFloat))` here to be on the safe side?